src/objects.rs

```diff
@@ -76,11 +76,11 @@ pub struct Blueprint {
     #[serde(skip_serializing_if = "Vec::is_empty")]
     pub schedules: Vec<Schedule>,
     #[serde(skip_serializing_if = "Option::is_none")]
-    pub position_relative_to_grid: Option<Position>,
-    #[serde(skip_serializing_if = "Option::is_none")]
     pub snap_to_grid: Option<Position>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub absolute_snapping: Option<bool>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub position_relative_to_grid: Option<Position>,
```

Comment:
Please undo non-semantic changes here, where you moved the location of `position_relative_to_grid`.